### ISSUE TITLE
[BITV]: Create possibility to change a color theme for native datetime picker on dark theme

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42392]
For now a picker itself is not visible on dark theme. This have to be changed.

![Screenshot from 2023-12-20 10-58-33](https://github.com/nextcloud/server/assets/6078378/03516e96-b122-4b30-a230-afb35774d832)

Have to be changed to:

![Screenshot from 2023-12-20 11-00-10](https://github.com/nextcloud/server/assets/6078378/ab62ec92-1896-4480-9c03-9d2ce2b23959)

**Comment by szaimen** [comment]
I checked on 28 and it doesnt seem to be working yet:
![image](https://github.com/nextcloud/server/assets/42591237/f3e1c43a-64f0-4117-a200-8b8e4bf8b8ee)
(also the icon is not visible)

**Comment by szaimen** [comment]
The problem seems to be that I am using system default theme: 
![image](https://github.com/nextcloud/server/assets/42591237/8950363d-28aa-4c60-96d9-fa9e6f5b725a)

**Comment by JuliaKirschenheuter** [comment]
![image](https://github.com/nextcloud/server/assets/6078378/83ef76f2-ffc2-4b6f-86fa-c1ceff08965b)

**Comment by emoral435** [comment]
Moved to Parked, as we will need to wait to get stable28's library bumped :)

**Comment by szaimen** [comment]
Still doesnt seem to work with follow system theme on stable 28 😢
![image](https://github.com/nextcloud/server/assets/42591237/f7840a94-82f4-4fa2-8fa3-98e816a08ead)

**Comment by JuliaKirschenheuter** [comment]
can be closed when https://github.com/nextcloud/server/pull/43485 is merged
